src/components/dialogs/move-dialog.tsx

```diff
@@ -14,10 +14,17 @@ interface MoveDialogProps {
     open: boolean;
     onClose: (selectedDir: TreeViewFinderNodeProps[]) => void;
     itemsCount: number;
-    titleId?: string;
+    isDirectoryMoving?: boolean;
+    directoryName?: string;
 }
 
-const MoveDialog: FunctionComponent<MoveDialogProps> = ({ open, onClose, itemsCount, titleId }) => {
+const MoveDialog: FunctionComponent<MoveDialogProps> = ({
+    open,
+    onClose,
+    itemsCount,
+    isDirectoryMoving,
+    directoryName,
```

Comment:
If you pass `validationButtonText` and `title` props here, you don't add `*directory*` props here to a generic component `MoveDialog` which are relative to only one use case (move directory instead of elements).